src/axios.ts

```diff
@@ -3,12 +3,19 @@ import http from 'http';
 import https from 'https';
 import emitter from './emitter';
 
+// For a compatibility with Snyk CLI
+export declare interface Global extends NodeJS.Global {
+  ignoreUnknownCA: boolean;
+}
+declare const global: Global;
+
 const agentOptions = {
   keepAlive: true,
   maxSockets: 100, // Maximum number of sockets to allow per host. Defaults to Infinity.
   maxFreeSockets: 10,
   freeSocketTimeout: 60000, // // Maximum number of sockets to leave open for 60 seconds in a free state. Only relevant if keepAlive is set to true. Defaults to 256.
```

Comment:
@Arvi3d I am fairly sure this settings does nothing here 😇 It is not supported at https://nodejs.org/docs/latest-v12.x/api/http.html#http_new_agent_options and it looks like it is used only, when https://www.npmjs.com/package/agentkeepalive is around. I'd vote for removing it and typing agentOptions properly.